contracts/NonfungibleTokenPositionDescriptor.sol

```diff
@@ -11,7 +12,21 @@ import './libraries/NFTDescriptor.sol';
 
 /// @title Describes NFT token positions
 /// @notice Produces a string containing the data URI for a JSON metadata string
-contract NonfungibleTokenPositionDescriptor is INonfungibleTokenPositionDescriptor {
+contract NonfungibleTokenPositionDescriptor is INonfungibleTokenPositionDescriptor, Ownable {
```

Comment:
idt this should be ownable

Reply:
If we want governance to be able to add more tokens, some sort of ownable is necessary, but if we don't want to add more tokens in the future, then we can take this out